MadWorkshopFunction/src/main/java/app/App.java

```diff
@@ -19,6 +19,11 @@
 public class App implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
 
     public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEvent input, final Context context) {
+        try {
+            final com.amazonaws.services.s3.AmazonS3 s3Client = 
+                com.amazonaws.services.s3.AmazonS3ClientBuilder.standard().withRegion("ap-northeast-1").build();
+            final com.amazonaws.services.s3.model.ObjectListing files = s3Client.listObjects("dummy-bucket");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses an outdated API. [ListObjectsV2 is the revised List Objects API, and we recommend you use this revised API for new application developments](https://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html).